test/SILOptimizer/Mirror-LoadableByAddress-failure.swift

```diff
@@ -0,0 +1,35 @@
+//===--- Mirror.swift -----------------------------------------------------===//
+//
```

Comment:
Can you move this file from the SILOptimizer sub-folder to the IRGen one? this is an IRGen test